doc/api/zlib.md

```diff
@@ -396,60 +399,61 @@ Provides an object enumerating Zlib-related constants.
 added: v0.5.8
 -->
 
-Returns a new [Deflate][] object with an [options][].
+Passes [`options`][] to the `Deflate` constructor and returns a new [`Deflate`][] object.
 
 ## zlib.createDeflateRaw([options])
 <!-- YAML
 added: v0.5.8
 -->
 
-Returns a new [DeflateRaw][] object with an [options][].
+Passes [`options`][] to the `DeflateRaw` constructor and returns a new [`DeflateRaw`][] object.
 
 ## zlib.createGunzip([options])
 <!-- YAML
 added: v0.5.8
 -->
 
-Returns a new [Gunzip][] object with an [options][].
+Passes [`options`][] to the `Gunzip` constructor and returns a new [`Gunzip`][] object.
 
 ## zlib.createGzip([options])
 <!-- YAML
 added: v0.5.8
 -->
 
-Returns a new [Gzip][] object with an [options][].
+Passes [`options`][] to the `Gzip` constructor and returns a new [`Gzip`][] object.
 
 ## zlib.createInflate([options])
 <!-- YAML
 added: v0.5.8
 -->
 
-Returns a new [Inflate][] object with an [options][].
+Passes [`options`][] to the `Inflate` constructor and returns a new [`Inflate`][] object.
 
 ## zlib.createInflateRaw([options])
 <!-- YAML
 added: v0.5.8
 -->
 
-Returns a new [InflateRaw][] object with an [options][].
+Passes [`options`][] to the `InflateRaw` constructor and returns a new [`InflateRaw`][] object.
 
 ## zlib.createUnzip([options])
 <!-- YAML
 added: v0.5.8
 -->
 
-Returns a new [Unzip][] object with an [options][].
+Passes [`options`][] to the `Unzip` constructor and returns a new [`Unzip`][] object.
 
 ## Convenience Methods
 
 <!--type=misc-->
 
-All of these take a [Buffer][] or string as the first argument, an optional
-second argument to supply options to the `zlib` classes and will call the
-supplied callback with `callback(error, result)`.
+All of these take a [Buffer][] or string as the first argument and an optional 
+second argument to supply options to the `zlib` classes.
+
+All the methods have asynchronous and synchronous forms. The asynchronous 
+form always takes a completion callback as its last argument in
+the form `callback(error, result)`.
 
-Every method has a `*Sync` counterpart, which accept the same arguments, but
-without a callback.
```

Comment:
This change was just to align it with the equivalent note about sync/async methods in `fs` which is currently:

> All the methods have asynchronous and synchronous forms.
> 
> The asynchronous form always takes a completion callback as its last argument. The arguments passed to the completion callback ...

So the `zlib` text for the same would be (with the change in this PR):

> All the methods have asynchronous and synchronous forms. The asynchronous form always takes a completion callback as its last argument in the form callback(error, result).

Note that just below this on the page the actual sync and non-sync methods are listed explicitly.

Does that change things? Otherwise, I can revert it to how it was.